recommenders/__init__.py

```diff
@@ -24,10 +24,6 @@
 
 
 
-
-movies_df = pd.read_csv("movies.csv")
-ratings_df = pd.read_csv("ratings.csv")
-
 '''
 Uncomment the below line after you complete the code above.

```

Comment:
The commented block of hash marks on line 9 is unnecessary and should be removed to improve readability.